core/src/main/java/com/linecorp/armeria/client/proxy/ProxyConfigSelector.java

```diff
@@ -67,16 +68,27 @@ void connectFailed(SessionProtocol protocol, Endpoint endpoint,
                        SocketAddress sa, Throwable throwable);
 
     /**
-     * Provides a way to re-use an existing {@link ProxySelector} with some limitations:
-     * 1. Some incompatibilities when used with sun's {@code DefaultProxySelector}
-     *     - Some properties like socksProxyVersion aren't respected
-     *     - This class doesn't attempt to resolve scheme format differences.
-     *       For instance, sun's {@code DefaultProxySelector} requires basic scheme formats "http", "https".
-     *       However, armeria uses scheme formats including serialization format ("none+http", "tbinary+h1c").
-     *       This may be a source of unexpected behavior.
-     * 2. Selecting multiple {@link Proxy} isn't supported.
+     * Provides a way to re-use an existing {@link ProxySelector} with some limitations.
+     * <ul>
+     *     <li>Incompatibilities when used with JDK's default {@link ProxySelector} implementation:
+     *       <ul>
```

Comment:
- re-use -> reuse
- Inconsistent indentation. Could you use 2-space indentation for HTML tags?